metricbeat/module/ceph/health/_meta/fields.yml

```diff
@@ -0,0 +1,69 @@
+- name: health
+  type: group
+  description: >
+    health
+  fields:
+    - name: cluster.overall_status
+      type: keyword
+      description: >
+        Overall status of the cluster
+    - name: cluster.timechecks.epoch
+      type: long
+      description: >
+        Map version
+    - name: cluster.timechecks.round
+      type: long
+      description: >
+        timecheck round
+    - name: cluster.timechecks.round_status
+      type: keyword
+      description: >
+        Status of the round
+    - name: mon.avail_percent
```

Comment:
I would change this to `mon.available.pct`. See https://www.elastic.co/guide/en/beats/libbeat/5.1/event-conventions.html